Failing to build

Not very familiar with Java, so this could be my fault. I'm trying to build this, and edited the following section of pom file:

```
        <dependency>
            <groupId>net.opentsdb</groupId>
            <artifactId>opentsdb</artifactId>
            <version>2.0.0</version>
            <scope>system</scope>
            <systemPath>/home/vagrant/opentsdb-2.0.0/build/tsdb-2.0.0.jar</systemPath>
        </dependency>
```

However, when I try to build I get `ElasticSearch is not abstract and does not override abstract method collectStats(StatsCollector) in SearchPlugin`:

Full maven output:

```
[vagrant@localhost opentsdb-elasticsearch]$ mvn clean install
[INFO] Scanning for projects...
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for net.opentsdb:otsdb-elasticsearch:jar:1.0
[WARNING] 'dependencies.dependency.systemPath' for net.opentsdb:opentsdb:jar should use a variable instead of a hard-coded path /home/vagrant/opentsdb-2.0.0/build/tsdb-2.0.0.jar @ line 56, column 14
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-source-plugin is missing. @ line 97, column 21
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING] 
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building otsdb-elasticsearch 1.0
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ otsdb-elasticsearch ---
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ otsdb-elasticsearch ---
[WARNING] Using platform encoding (UTF-8 actually) to copy filtered resources, i.e. build is platform dependent!
[INFO] skip non existing resourceDirectory /home/vagrant/opentsdb-elasticsearch/src/main/resources
[INFO] 
[INFO] --- maven-compiler-plugin:2.5.1:compile (default-compile) @ otsdb-elasticsearch ---
[WARNING] File encoding has not been set, using platform encoding UTF-8, i.e. build is platform dependent!
[INFO] Compiling 2 source files to /home/vagrant/opentsdb-elasticsearch/target/classes
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /home/vagrant/opentsdb-elasticsearch/src/net/opentsdb/search/ElasticSearch.java:[59,13] error: ElasticSearch is not abstract and does not override abstract method collectStats(StatsCollector) in SearchPlugin
[INFO] 1 error
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 2.793s
[INFO] Finished at: Fri Sep 13 14:43:06 UTC 2013
[INFO] Final Memory: 11M/27M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:2.5.1:compile (default-compile) on project otsdb-elasticsearch: Compilation failure
[ERROR] /home/vagrant/opentsdb-elasticsearch/src/net/opentsdb/search/ElasticSearch.java:[59,13] error: ElasticSearch is not abstract and does not override abstract method collectStats(StatsCollector) in SearchPlugin
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```

--- a/src/net/opentsdb/search/ElasticSearch.java
+++ b/src/net/opentsdb/search/ElasticSearch.java
@@ -47,6 +47,7 @@ import net.opentsdb.meta.Annotation;
 import net.opentsdb.meta.TSMeta;
 import net.opentsdb.meta.UIDMeta;
 import net.opentsdb.search.SearchQuery.SearchType;
+import net.opentsdb.stats.StatsCollector;
 import net.opentsdb.utils.JSON;
 
 import com.fasterxml.jackson.core.JsonParseException;
@@ -251,6 +252,13 @@ public final class ElasticSearch extends SearchPlugin {
   /** @return the version of this plugin */
   public String version() {
     return "2.0.0";
+  }
+
+  /**
+   * Called by the TSD when a request for statistics collection has come in.
+   * Method stubbed, no statistics are available
+   */
+  public void collectStats(final StatsCollector collector) {
   }
   
   /**

